docgen/src/examples/e-commerce-infinite/App.js

```diff
@@ -160,7 +120,8 @@ const ColorItem = ({item, createURL, refine}) => {
 };
 
 const CustomColorRefinementList = ({items, refine, createURL}) =>
-    <div>
+    items.length > 0 ? <div>
+      <h5 className={'ais-Panel__Title'}>Colors</h5>
```

Comment:
I'm not sure that having a `connectPanel` would be easier than just checking for the `items` length (or any condition that works for you).

Also I'm not sure to see an easy way to hide the fact that in order for the panel to work, you need to notify it when your condition is met.